content/en/blog/2021-09-29-server-side-reconciliation-is-coming.md

```diff
@@ -0,0 +1,190 @@
+moved [server-side
+apply](https://kubernetes.io/docs/reference/using-api/server-side-apply/)
+to GA, we are offering you a new reconciler based on it, and graduating
+the API to `v1beta2`.
+
+## What's happening
+
+- **When does this happen?**\
+  With the release of Flux 0.18, we will move to the new reconciler.
+  It will be released in the coming weeks. Refer to [this
+  PR](https://github.com/fluxcd/kustomize-controller/pull/426)
+  for more information.
+
+- **Do I have to use the new thing?**\
+  Yes. Flux will be more performant, less error-prone and from a
+  maintenance perspective will be a lot easier for us. We understand
+  that this new feature will require changes on your end, but we are
+  certain you are going to like the new experience!
+
+- **Will my clusters stop working?**\
+  No, but you will need to do a little preparation to make sure Flux
+  can still apply your configurations. See below.\
+  *Note:* The pre-flight checks should be able to catch issues like meeting the
+  minimum required Kubernetes version.
+
+## Here is what you get
+
+- The new reconciler improves performance (CPU, memory, network, FD
+  usage) and reduces the number of calls to Kubernetes API by
+  replacing `kubectl exec` calls with a specialized applier written in
+  Go.
+- We are able to validate and reconcile sources that contain both CRDs
+  and CRs.
+- Detects and reports drift between the desired state (git, s3, etc)
+  and cluster state reliably.
+- In the future: Preview of local changes to manifests without pushing
+  to upstream (`flux diff -k` command TBA).
+- Being able to wait for all applied resources to become ready without
+  requiring users to fill in the health checks list.
+- Improves the overall observability of the reconciliation process by
+  reporting in real-time the garbage collection and health
+  assessment actions.
+
+## This is what you need to do to prepare
+
+**Check the Kubernetes version you are running in your cluster.**
+All the versions below fix a regression in the [managed fields and field
+type](https://github.com/kubernetes/kubernetes/pull/91748).
+
+| Kubernetes version | Minimum required |
+| --- | --- |
+| `v1.16` | `>= 1.16.11` |
+| `v1.17` | `>= 1.17.7` |
+| `v1.18` | `>= 1.18.4` |
+| `v1.19` and later | `>= 1.19.0` |
+
+**Namespaced objects must contain metadata.namespace, defaulting to the
+default namespace is no longer supported**. This means you will need to
+chase down any namespaced resources in your configuration files that are
+left to default, and give them a namespace. Keep in mind that
+kustomizations are often used to assign a namespace, so even if a
+particular file doesn't have a namespace in it, it may not represent a
+problem.
+
+**The logs, events and alerts that report Kubernetes namespaced object
+changes are now using the `Kind/Namespace/Name` format instead of
+`Kind/Name`** e.g.:
+
+```cli
+Service/flux-demo/podinfo unchanged\
```

Comment:
Please remove the `\` from here and the line below.

Reply:
Will do - sorry - that's an artifact from `pandoc` converting from `docx` (Google doc) to `markdown`.